src/Microsoft.DotNet.Tools.Compiler/project.json

```diff
@@ -1,31 +1,33 @@
 ﻿{
-  "name": "dotnet-compile",
```

Comment:
Yep, four spaces is the default AFAIK. It's certainly what I have on mine and I don't think I've ever changed it...